webpack/envs/sharedConfig.js

```diff
@@ -9,12 +9,23 @@ export const productionDevtool = env.webpackDevtool || "source-map"
 export const mode = env.webpackDebug ? "development" : env.nodeEnv
 export const stats = env.webpackStats || "errors-only"
 
+export const experiments = {
+  lazyCompilation: {
+    entries: false,
+    imports: false,
+  },
```

Comment:
Note this flag is currently off. There's some rough edges around lazy compilation and loadable components / bundle-splitting, but will continue looking into a fix here.